data/json/recipes/other/vehicles.json

```
@@ -1505,5 +1505,41 @@
     "book_learn": [ [ "textbook_computer", 1 ], [ "advanced_electronics", 1 ], [ "reference_fabrication1", 1 ] ],
     "using": [ [ "3d_printing_standard", 10 ] ],
     "components": [ [ [ "carbon_fiber_filament", 100 ] ] ]
+  },
+  {
+    "result": "frame_plastic",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "computer",
+    "difficulty": 1,
+    "autolearn": true,
+    "time": "20 m",
+    "using": [ [ "3d_printing_standard", 20 ] ],
+    "components": [ [ [ "plastic_chunk", 20 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "frame_plastic",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "40 m",
+    "autolearn": true,
+    "tools": [ [ [ "mold_plastic", -1 ] ], [ [ "surface_heat", 30, "LIST" ] ] ],
+    "components": [ [ [ "plastic_chunk", 20 ] ] ]
```

Compare standard plastic boat hull using 50 doses of `surface_heat` and taking 1.5 times longer than the comparable printer recipe.
```suggestion
    "time": "75 m",
    "autolearn": true,
    "tools": [ [ [ "mold_plastic", -1 ] ], [ [ "surface_heat", 75, "LIST" ] ] ],
    "components": [ [ [ "plastic_chunk", 50 ] ] ]
```